library/core-components/components/hero/hero.tsx

```diff
@@ -1,72 +1,114 @@
-import React from 'react';
-import { cx } from '@emotion/css';
+import React, { forwardRef } from 'react';
+import { ArrowRight } from '@rangle/radius-foundations/generated/icons';
+import { radiusTokens } from '@rangle/radius-foundations/generated/design-tokens.constants';
 
 import { RadiusAutoLayout } from '../../components/auto-layout/auto-layout';
 import { RadiusButton } from '../../components/button/button';
 import { Typography } from '../../components/typography/typography';
-import { getStyles } from './hero.styles';
+import { useStyles } from './hero.styles';
 
-export type HeroProps = {
-  title: string;
+export type RadiusHeroProps = {
+  /** The main header text */
+  header: string;
+  /** The eyebrow text */
   eyebrow: string;
-  buttonLabel: string;
+  /** The image source */
   imageSrc: string;
-  ctaUrl: string;
-} & React.HTMLAttributes<HTMLDivElement>;
+  /** The image alt text */
+  imageAlt: string;
+  /** The button label text. If not provided, the button will not be shown */
+  buttonLabel?: string;
+  /** The url for the Call To Action link button. If not provided, the button will not be shown */
+  ctaUrl?: string;
+  className?: string;
+};
 
-export const RadiusHero = ({
-  title,
-  eyebrow,
-  buttonLabel,
-  imageSrc,
-  ctaUrl,
-  className,
-  ...rest
-}: HeroProps) => {
-  const heroStyle = getStyles();
-  return (
-    <RadiusAutoLayout className={cx(heroStyle, className)} {...rest}>
+/**
+ * The Hero component is used to display a large image with a header, eyebrow,
+ * and optional CTA button.
+ *
+ * If either the `ctaUrl` or `buttonLabel` prop are not provided, the Call To
+ * Action link button will not be shown.
+ *
+ * ### Resources
+ * [Figma Design Specs](https://www.figma.com/file/hW6DtJyTGtGlLkRB3jK68v/Booster---Hero?node-id=2-2)
+ */
+export const RadiusHero = forwardRef<HTMLDivElement, RadiusHeroProps>(
+  (
+    { header, eyebrow, buttonLabel, imageSrc, imageAlt, ctaUrl, className },
+    ref
+  ) => {
+    const { contentContainer, imageContainer, textContainer } = useStyles();
+    return (
       <RadiusAutoLayout
-        width="fill-parent"
-        // @ts-expect-error TODO: fix this when we have the correct component tokens
-        space="--spacing-core-space-6x"
-        alignment="center"
-        className="content-container"
+        className={className}
+        ref={ref}
+        padding={[
+          radiusTokens.component.spacing.hero.padding.vertical,
```

Comment:
Small suggestion - since we typically will just use component tokens, maybe it makes sense to have a separate object which can be imported so these can be shorter. (the original object doesn't need to be modified)